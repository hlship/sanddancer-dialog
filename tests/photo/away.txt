brood stories
open wallet
brood job
n
drop photo
n
purloin key
brood last day
brood ocean
s
unfold photo
fold photo
unfold photo