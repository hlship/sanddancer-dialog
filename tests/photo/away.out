

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> brood stories
There are dark spirits who roam the earth, little Knock. Grandma used to say
that, holding you tight and stroking your hair. There are dark spirits who roam
the earth, but you're not alone. Oh, no. I'm here. (She'd kiss your head and
you'd squeeze her back.) But others are watching out for you too. You have three
animal guardians, hmm? Spirits who are always watching over you. Oh, you can't
always trust them to know what's best. Remember that, Knock. But when you need
help, they'll come, and protect you from the worser things in the world.

Mom would yell at Grandma a lot for filling your head with that new-age
bullshit. Grandma grew up white and midwestern and Baptist, but had started
wearing things with feathers and playing the pan flute by the time you were
born. She seemed to really like having a son-in-law who was Native American or
American Indian or Indigenous People or whatever she'd decided the term was that
week, and she was pretty pissed when Mom left him. Anyway. Her stories were
mostly BS, you guess, but some of them stuck with you. When it's dark you still
wonder if your spirit animals are out there, somewhere, and what the hell is
taking them so long to find you.

> open wallet
You open your wallet, revealing your driver's license, a receipt from Big
Jimmy's, and a folded photo.

Something about the receipt from Big Jimmy's makes you want to brood about your
shit job.

> brood job
Juza straddles the 371 like a drunk at last call, smelly and without a plan for
the future. Shiny cars whiz by at ninety once in a while, only stopping for gas
or directions (hint: not this way). No one ever stops for the stalls selling
food and jewelry and blankets and cheap t-shirts that cluster around the dirt
turn-off onto the rez. At least, no one buying anything.

Big Jimmy shook his head when he heard you lived in Oro Oeste. Hell of a drive,
kid, he said, popping up his Lumberjacks cap to scratch the straw pate
underneath. You should get a place in Hoo-zuh. Little Jimmy'll set you up in
that trailer cross the road for almost nothin'.

Almost nothing. Yeah, exactly what you'd turn into on the cold day in hell you
move to Juza. But on the other hand, that was before you met Ocean. Now the road
back to Oro Oeste seems longer and longer every night.

You shiver in the cold.

> n
(first attempting to get out of the pickup truck)
The door creaks open with difficulty, rusted joints straining. You get out of
the pickup truck.

You walk north.

Crumbling Concrete
This concrete building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned: a pane of cracked glass sparkles
faintly in the beam of your headlights. Tendrils of desert sand drift against a
boarded-up door to the northeast leading inside; you could also walk around to
the east or head south back to your truck.

> drop photo
(first attempting to take the photo)
You take the photo out of your wallet.

The photo falls to the ground.

> n
You walk north.

Base of the Tower
Behind the concrete building a steel girder rises from the sand, one of three
legs of the huge electrical tower looming like some gargantuan spider into
blackness above you. A tumbleweed drifts lazily against the rusted metal among
scrawny weeds and bits of trash. Someone left a tin can here years ago: ridged
ripples run down its rusty side.

The desert stretches in all directions except back south towards the building.

Something about the tumbleweed makes you want to brood about your last day of
high school.

> purloin key

> brood last day
You didn't expect it was going to be your last day. But that morning you got
called in to the principal's office and fat bald Mr. Cox and pissy old Mrs.
Burke were there, and they looked kind of like strong animals stalking weak
animals, and you knew something bad was up.

Cox said I have here, Mr. Morales (a bad start since you hate that name) a test
you took last week in Mrs. Burke's sophmore English class. Questions have been
raised (he looked up at Mrs. Burke like he was trying to pass the buck)
questions have been raised about the quality of your essay, and whether a
student with your academic and behavioral record (he scratched his bloated nose
meaningfully) could have plausibly produced such as essay, and you get the idea.
They thought you cheated.

No. They knew you cheated, deep in their smug empty hearts. The wanted you to
admit it, say you were a cheater and a liar. But you weren't. You wrote the
essay, every god damn word, because you really liked the book for once and
wanted to show Mrs. Burke that maybe if they gave people better books to read
kids would actually learn something. But they wanted a confession. They wanted a
thieving example they could parade in front of the school. Someone of your
academic and behavioral and economic and racial background and yeah, screw this
shit. So you got up and left and never came back. Drop out, hell. You walked out
and you'd do it again.

Something about the rusted key makes you want to brood about meeting Ocean.

> brood ocean
She was buying a Fresca at Big Jimmy's when you got off your shift, shoving work
keys in your pocket, and you stood behind her waiting to get some cigs, almost
too tired to notice how cute she was but still noticing, yeah, still noticing.

She turned around and caught you noticing and you were pretty embarrassed and
covered in sweat and grease from the garage, so you stepped up and bought some
Camels and were pretty surprised that she was still there when you turned
around.

She told you her name was Ocean Running Deer and she lived on the rez and you
told her your name was Nakaibito Morales and you lived in Oro Oeste. She said
isn't Nakaibito the name of a town off west and you told her your mom had picked
it off a map because it sounded like a good Indian name. You never tell anyone
that story but for some reason you told her. She laughed, sweetly though, and
said maybe the two of you should go there sometime and see what is was like.
Somehow two weeks later you were dating although you never ended up going to
Nakaibito.

And now... no, you can't think about that yet.

> s
You walk south.

Crumbling Concrete
This concrete building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned: a pane of cracked glass sparkles
faintly in the beam of your headlights. Tendrils of desert sand drift against a
boarded-up door to the northeast leading inside; you could also walk around to
the east or head south back to your truck.

You see a folded photo here.

> unfold photo
You unfold the photo revealing the ultrasound of Ocean's baby. Your baby.

The air is frigid, and turns your breath into clouds.

Something about the ultrasound makes you want to brood about when you heard the
news.

> fold photo
You fold the photo.

> unfold photo
You unfold the photo revealing the ultrasound of Ocean's baby. Your baby.

> 
